include/can/simlib/socketcan_transport.hpp

```diff
@@ -10,6 +10,7 @@
 #include <unistd.h>
 
 #include <cstring>
```

Comment:
do you still need to include the cstring lib?

Reply:
yeah the socket setup uses `strcpy`